docs/calicoctl/resources/bgppeer.md

```diff
@@ -37,4 +37,4 @@ spec:
 #### Spec
 | name     | description                 | requirements               | schema  |
 |----------|-----------------------------|----------------------------|---------|
-| asNumber | The AS Number of this peer. | Must be a valid AS Number. | integer |
+| asNumber | The AS Number of this peer. | Must be a valid AS Number.  The YAML or JSON format may be a string and may be specified in dotted notation. | integer |
```

Comment:
Does this work in markdown?  In RST, you have to change the length of all the other lines in the table.